src/client/components/pages/entity-revisions.js

```diff
@@ -16,6 +16,7 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+
```

Comment:
It is usually better to keep code style changes (such as whitespace) in separate commits (or even completely separate PRs), so they can be reviewed outside of "actual" code changes.